.github/workflows/test_build_and_deploy.yml

```diff
@@ -26,11 +26,12 @@ jobs:
         run: |
           echo "DOMAIN=${{env.DOMAIN}} 
           EXTERNAL_IP=${{secrets.SANDBOX_HOST}} 
-          TAG=${{env.TAG}}
```

Comment:
As far as I understand, now we always deploy on new `edge` which is basically new commit on `master`, is this intended?

Reply:
Yup, this is intended to use the newest jellyfish version.